ursh/indico_ursh/controllers.py

```diff
@@ -20,6 +20,7 @@
 
 import requests
 from flask import jsonify, request
+from indico_ursh import _
```

Comment:
import order; imports from the current package go in the last block of imports (ie a new one below the `indico.*` imports)